pkg/rbac/messagesDev.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Contributors to the Open Cluster Management project
+
```

Comment:
@jlpadilla What tags should be here? Is it same as https://github.com/stolostron/search-indexer/blob/main/pkg/config/config_development.go#L3-L7

Reply:
Maybe `test`?  You'll also need to update the Makefile to include this tag like here.
https://github.com/stolostron/search-indexer/blob/main/Makefile#L13-L14

Reply:
Also, try naming the file `*_test.go`.  That has a similar effect, but not sure if it works across packages.

Reply:
Done adding tags.